build.gradle

```diff
@@ -183,6 +183,16 @@ task rcaTest(type: Test) {
     testLogging.showStandardStreams = true
 }
 
+task rcaIt(type: Test) {
+    dependsOn checkstyleMain
+    dependsOn checkstyleTest
+    dependsOn spotbugsMain
+    useJUnit {
+        includeCategories 'com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.integTests.framework.RcaItMarker'
+    }
+    //testLogging.showStandardStreams = true
```

Comment:
remove this line ?

Reply:
Wanted to keep it so than people can enable it if they want stdout ?